docs/semgrep-supply-chain/triage-remediation.md

```diff
@@ -40,33 +40,20 @@ Perform triage and remediation on your open source dependencies through the **Su
 <dd>This tab displays information about all of your dependencies across all onboarded repositories.</dd>
 </dl>
 
-![Semgrep Supply Chain Vulnerabilities page](/img/sc-vulnerabilities.png)
+![Semgrep Supply Chain Vulnerabilities page](/img/sc-vulns.png)
 _Figure 1_. Semgrep Supply Chain Vulnerabilities page.
 
-## Assessing and triaging dependency findings and usages
+## Assess and triage dependency findings and usages
 
 :::info Prerequisite
 At least one repository that scans for dependencies through Semgrep Supply Chain. See [Getting started with Semgrep Supply Chain](/semgrep-supply-chain/getting-started).
```

Comment:
Link text (title) needs updating to "scan third-party dependencies" (http://semgrep.dev/docs/semgrep-supply-chain/getting-started)